libnetwork/osl/namespace_linux.go

```diff
@@ -171,7 +181,7 @@ func GenerateKey(containerID string) string {
 			indexStr string
 			tmpkey   string
 		)
-		dir, err := os.ReadDir(basePath())
+		dir, err := ioutil.ReadDir(basePath())
```

Comment:
Looks like this is reverting a recent change (bad rebase perhaps?) the master branch uses `os` (the 20.10 branches uses the old `ioutil`)